vue/components/form-input/form-input.stories.js

```diff
@@ -0,0 +1,94 @@
+import { storiesOf } from "@storybook/vue";
+import { withKnobs, text, number, select } from "@storybook/addon-knobs";
+
+storiesOf("Molecules", module)
+    .addDecorator(withKnobs)
+    .add("Form Input", () => ({
+        props: {
+            variant: {
+                default: select(
+                    "Variant",
+                    {
+                        Unset: null,
+                        Inline: "inline"
+                    },
+                    null
+                )
+            },
+            header: {
+                default: text("Header", "Header")
+            },
+            footer: {
+                default: text("Footer", "Footer")
+            },
+            headerVariant: {
+                default: select(
+                    "Header Variant",
+                    {
+                        Unset: null,
+                        Error: "error",
+                        Warning: "warning",
+                        Success: "success"
+                    },
+                    null
+                )
+            },
+            headerSize: {
+                default: select(
+                    "Header Size",
+                    {
+                        Unset: null,
+                        Small: "small",
+                        Medium: "medium",
+                        Large: "large"
+                    },
+                    "medium"
+                )
+            },
+            footerVariant: {
+                default: select(
+                    "Footer Variant",
+                    {
+                        Unset: null,
+                        Error: "error",
+                        Warning: "warning",
+                        Success: "success"
+                    },
+                    null
+                )
+            },
+            footerSize: {
+                default: select(
+                    "Footer Size",
+                    {
+                        Unset: null,
+                        Small: "small",
+                        Medium: "medium",
+                        Large: "large"
+                    },
+                    "small"
+                )
+            },
+            headerMinWidth: {
+                default: number("Header Minimum Width", null)
+            },
+            footerMinWidth: {
+                default: number("Footer Minimum Width", null)
+            }
+        },
+        template: `
+            <form-input
+                v-bind:variant="variant"

```

Comment:
when variant is inline, the top border gets cutted
<img width="1235" alt="imagem" src="https://user-images.githubusercontent.com/24736423/74862681-7be97380-5344-11ea-8546-a063f1d62287.png">